### PR TITLE
Task 11 and 13

### PR DESCRIPTION
**Pull request by navjot-chahal**
@pmvald I was under the impression that the grid should be open at the border and it must implement. So I masked the outer grid section similar to what "client.html" does in the canvas theme reference.

I tried to use the nth-child selector similar to what you mentioned with borders on the right and the bottom and no right borders for every (6n+6) child (right-most grid item). But being a bootstrap grid the elements inside are cascaded in a row-based flexbox. So there are no special attributes present in the last row logos that we can use in the nth-child selector to have its bottom-border removed.

I assume that we wanted this layout to work for any number of logos, so that's why my code was very long previously. But for this pull in order to use nth-child, I had to hard code the "border-bottom: none" for the last four logos. This is not what I initially thought, as this works for 16 logos (given). Please advise if this is acceptable or if there is a better way to implement this with nth-child.

PS: Thanks for your patience.

### CODE REVIEW

**Comment by navjot-chahal** [comment]
@pmvald I have added task 13 additions to this pull request as well. Please review my work.

File: -sass/styles/_collapse.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    }
+  }
+
+  &__symbol {
+    position: absolute;
+    transition-property: transform;
+  }
+}
+
+@keyframes collapse-animation {
```
Comment (pmvald) [inline -sass/styles/_collapse.scss:24]: good

File: -sass/styles/_dropdown.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.dropdown {
+  &__content {
+    overflow: hidden;
```
Comment (pmvald) [inline -sass/styles/_dropdown.scss:3]: maybe this should also move to the layouts

File: -sass/styles/_dropdown.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    top: 2rem;
+
+    &--open {
+      animation: dropdown-animation 0.5s ease-in forwards;
+    }
+  }
+  &__toggle {
+    height: 1rem + $menu-link-padding-x;
```
Comment (pmvald) [inline -sass/styles/_dropdown.scss:11]: we usually don't specify the height directly. You may add padding top/bottom - the height becomes whatever it has be

File: -sass/styles/_dropdown.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    @include caret();
```
Comment (pmvald) [inline -sass/styles/_dropdown.scss:12]: where is caret defined?

File: -sass/styles/_collapse.scss [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    top: 2rem;
+
+    &--open {
+      animation: collapse-animation 0.5s ease-in forwards;
+    }
+  }
+
+  &__toggle {
+    height: 1rem + $menu-link-padding-x;
+    &--open > span {
```
Comment (pmvald) [inline -sass/styles/_collapse.scss:13]: instead of `span` it is more standard to use `:after`

**Review by pmvald — CHANGES_REQUESTED** [review]

**Comment by pmvald** [comment]
@navjot-chahal thanks for your work. A few things: 
1. we asked the interns to pick the tasks labelled "help wanted". These tasks are meant for interns and have more detailed and simplified instructions. You picked a task that by chance was not meant for intern. The dropdown task is going to employ web components later ... I recommend you choose a "help wanted" task and move step by step
2. When you pick a task you should notify us (by dropping a line maybe). I didn't know you were working on this task ... 
3. You included task 11 in this pull request. I already closed that task and pull request. Besides, we should never mix or combine tasks. Each task should have its own pull request. Also I see you added a label "help wanted" to this pull request which is not meant to be here. As a team we need to follow the same guideline and avoid doing random stuff (and this is even more important than programming skills itself)
4. nevertheless, I made a few code review comments so that you get some idea. But I recommend you pick a "help wanted" task and be inline with other interns

File: -sass/styles/_dropdown.scss (hunk 4 above)
Comment (navjot-chahal) [reply -sass/styles/_dropdown.scss:12]: Caret is defined in:
> /-sass/bootstrap5/mixins/_caret.scss

I added it in the same way as bootstrap5 dropdown

File: -sass/styles/_dropdown.scss (hunk 3 above)
Comment (navjot-chahal) [reply -sass/styles/_dropdown.scss:11]: Changed in Task 13 - Additions.

File: -sass/styles/_dropdown.scss (hunk 2 above)
Comment (navjot-chahal) [reply -sass/styles/_dropdown.scss:3]: Changed in Task 13 - Additions.

**Comment by navjot-chahal** [comment]
@pmvald Changes added as requested. I will make sure I will follow the instruction carefully.

**Pull request closed without merging**